renormalizer/mps/mps.py

```diff
@@ -274,14 +290,22 @@ def ground_state(cls, model: Model, max_entangled: bool):
                         nzeros = basis.sigmaqn.count(0)
                         qn = basis.sigmaqn[dof.split("_")[1]]
                         if qn == 0:
-                            ms[0, dof.split("_")[1], 0] = 1. / np.sqrt(nzeros)
+                            if normalize:
+                                # Todo: here the dof_name only supports XXX_0, XXX_1
```

Comment:
Maybe a quick fix is to use the index of `0` elements in `basis.sigmaqn`?

Reply:
I will submit a quick pull-request soon.